src/parkapi_sources/converters/bahn_v2/validators.py

```diff
@@ -137,3 +173,43 @@ def __post_init__(self):
                 return
         # If no capacity with type PARKING was found, we miss the capacity and therefore throw a validation error
         raise ValidationError(reason='Missing parking capacity')
+
+
+@validataclass
+class BahnBikeLockedParkingSiteInput:
```

Comment:
Same as Mappers: copy paste, and can be removed completely as a single source would work, too.

Reply:
Updated, thanks!